subsys/suit/provisioning/CMakeLists.txt

```diff
@@ -201,4 +201,8 @@ if(DEFINED CONFIG_SUIT_MPI_SOC_NRF54H20)
 include(soc/nrf54h20.cmake)
 endif() # CONFIG_SUIT_MPI_SOC_NRF54H20
 
+if(DEFINED CONFIG_SUIT_MPI_SOC_NRF9280)
+include(soc/nrf9280.cmake)
+endif() # CONFIG_SUIT_MPI_SOC_NRF9280
+
```

Comment:
the whole of this upwards should be indented by 2 additional spaces due to be in an if()

Reply:
Fixed.